tools/webhook-apicoverage/resourcetree/basictypekindnode.go

```diff
@@ -33,33 +33,39 @@ func (b *BasicTypeKindNode) getData() nodeData {
 	return b.nodeData
 }
 
-func (b *BasicTypeKindNode) initialize(field string, parent INode, t reflect.Type, rt *ResourceTree) {
+func (b *BasicTypeKindNode) initialize(field string, parent NodeInterface, t reflect.Type, rt *ResourceTree) {
 	b.nodeData.initialize(field, parent, t, rt)
 	b.values = make(map[string]bool)
 	b.nodeData.leafNode = true
 }
 
-func (b *BasicTypeKindNode) isNil(v reflect.Value) (bool, string) {
+func (b *BasicTypeKindNode) analyzeType(t reflect.Type) {
+	if t.Name() != t.Kind().String() {
+		b.possibleEnum = true
+	}
+}
+
+func (b *BasicTypeKindNode) string(v reflect.Value) string {
 	switch v.Kind() {
 	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
 		if v.Int() != 0 {
-			return false, strconv.Itoa(int(v.Int()))
+			return strconv.Itoa(int(v.Int()))
```

Comment:
can we use a v.String() instead? https://golang.org/pkg/reflect/#Value.String

Reply:
v.String() returns "<bool Value>" (e.g. of bool but same for others) hence cannot be used.